src/client/apps/edit/components/admin/components/article/button_medium.tsx

```diff
@@ -0,0 +1,20 @@
+import colors from "@artsy/reaction/dist/Assets/Colors"
+import { avantgarde } from "@artsy/reaction/dist/Assets/Fonts"
+import styled from "styled-components"
+
+export const ButtonMedium = styled.button.attrs<{ background?: string }>({})`
```

Comment:
Is it possible to use a palette button here?

Reply:
I've tried overriding the `Button` component using the `styled(Button)` trick, but this doesn't seem to override the predefined CSS:

```tsx
// src/client/apps/edit/components/admin/components/article/button_medium.tsx
export const ButtonMedium = styled(Button).attrs<{ hasChanged?: boolean }>({})`
  background: ${({ hasChanged }) => hasChanged && colors.redMedium} // this does not override the background color
`

// src/client/apps/edit/components/admin/components/article/article_publish_date.jsx
<ButtonMedium
  width="100%"
  onClick={this.onMediaReleaseChange}
  hasChanged={hasChanged}
  disabled={hasChanged}
>
  Update
</ButtonMedium>
```

The palette `Button` definitely looks simpler except we can't override the background color, but I'm curious as to what's causing the behaviour above.